configmaster/management/handlers/config_backup.py

```diff
@@ -399,9 +403,13 @@ def run_completed(cls):
         for device_type in DeviceType.objects.all():
             if not device_type.config_filter:
                 continue
-            filename = os.path.join(
-                Repository.objects.get(id=1).path,
-                'Meta', "{}_filter.txt".format(
+
+            path = os.path.join(Repository.objects.get(id=1).path, 'Meta')
+
+            if not os.path.exists(path):
+                os.mkdir(path)
+
+            filename = os.path.join(path, "{}_filter.txt".format(
```

Comment:
This doesn't seem to be related to the Junos device type. I would prefer something like this to be in a separate commit, to make it easy to revert specific changes.

Reply:
A separate fix would be squashed on merge, and it's a trivial fix. I agree on principle, but let's not be more catholic than the pope ;-)

Reply:
Squashing all commits is not the only merge option. Regarding the catholisism: agreed, this is why it is already approved :-)

Reply:
Agreed then :-)